dev/docker/cudf/Dockerfile

```diff
@@ -12,7 +10,8 @@ ENV PATH=$SPARK_HOME/bin:$PATH
 ENV CUDA_ARCHITECTURES=70
 
 WORKDIR /opt/gluten
-RUN ./dev/buildbundle-veloxbe.sh --run_setup_script=ON --build_arrow=ON --spark_version=3.4 --enable_gpu=ON && \
+RUN bash ./dev/buildbundle-veloxbe.sh --run_setup_script=OFF --build_arrow=ON --spark_version=3.4 --enable_gpu=ON && \
```

Comment:
The base docker image runs setup_centos.sh in velox, but gluten makes some change on it, I'm not sure if it can run well.

Reply:
yes, I have verified locally, compilation is ok